libs/packages/components/src/lib/search-result-list/search-result-list.component.ts

```diff
@@ -1,15 +1,23 @@
-import { Component, Input, ContentChild, TemplateRef } from '@angular/core';
+import {
+  Component,
+  Input,
+  ContentChild,
+  TemplateRef,
+  OnInit,
+  OnChanges
+} from '@angular/core';
 @Component({
   selector: 'sds-search-result-list',
   templateUrl: './search-result-list.component.html',
   styleUrls: ['./search-result-list.component.scss']
```

Comment:
remove unused file

Reply:
This has been taken care of.

Reply:
Are you sure you're pushing your changes? I still see this in your branch